graylog2-server/pom.xml

```diff
@@ -460,6 +460,11 @@
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka_2.11</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-streams</artifactId>
+            <version>1.0.1</version>
```

Comment:
Why not use the latest version here?
https://mvnrepository.com/artifact/org.apache.kafka/kafka-streams/1.1.0

Reply:
@joschi... @muralibasani and I worked on this together and we choose to upgrade to 1.0.1 as it was the easiest to do from 9.0.1 and also provided the most value. Upgrading to 1.1.0 requires some more code rewriting on the graylog side as more API calls changes and code is deprecated/refactored. While it would provide some value, I see it as something that can be done in a future PR.

Reply:
@pbr0ck3r I'd prefer to use the latest stable version right away instead of having multiple PRs.